### DISCUSSION TITLE
How to translate "Commander"?

### DISCUSSION THREAD

**Discussion by Duke-M-commits** [ReactiveDrop/reactivedrop_translations#202] [Q&A] [answered]
I need to find a translation for "Commander" and "Commanders".
As far as I know, when sombody mentions commander, the person speaking is talking to the player.
But in this case, on the loading screen, I'm not sure who "Commanders" should be. And I also don't know why they are starring. I never really played L4D, so I don't know what all of this means.

Reference: resource/basemodui_german.txt:2010
`L4D360UI_Loading_Starring` and `L4D360UI_Loading_Costarring`

Related to this discussion in the following pr:
https://github.com/ReactiveDrop/reactivedrop_translations/pull/200#pullrequestreview-1054442782

**Comment by BenLubar** [comment, marked as answer]
We don't actually use these because the lobbies happen after the player is connected unlike Left 4 Dead 2 where the lobbies happen before the server starts. The commander is indeed the player.

**Comment by Duke-M-commits** [comment]
I just reread some 'Commander' lines in the closecaption_*.txt and realized the following.
The commander doesn't seem to be the player. Look at this example:
`"[english]Command.co_m2_ob7"		"<clr:240,255,255>Commander: Okay, I'm gonna bring Bloodhound in low and hover. You climb across her back while she's waiting. Step lightly now. She's gotta bring you home at the end of this."`

That's Lt. Williams!

Ok, now I think I get it:
The Commander in `L4D360UI_Loading_Starring` and `L4D360UI_Loading_Costarring` is one or more of the characters which can be selected in the lobby (what Ben meant in his answer).
But it's not the same Commander mentioned in closecaption_*.txt, that one is Lt. Williams.

**Comment by Duke-M-commits** [comment]
@BenLubar I think it would be more clear if we give this commander in closecaption_*.txt his real name. 
I will replace 'Commander' with 'Lt. Williams'. But I think I will leave the already translated lines as they are and just give the current translators an heads-up about the small change. So they can choose if they want to retranslate.

**Reply by BenLubar** [reply]
Yeah, the player never talks in closed captions so all of those are "Lt. Williams" or "Pilot"
